rosserial_vex_cortex/src/ros_lib/logger.h

```diff
@@ -0,0 +1,51 @@
+#ifndef __CANNON_LOGGER_H__
+#define __CANNON_LOGGER_H__
```

Comment:
Include guards should be `PACKAGE_NAME_FILE_NAME_H` format. http://wiki.ros.org/CppStyleGuide